content/applications/websites/ecommerce/products.rst

```diff
@@ -4,266 +4,341 @@
+     :guilabel:`Storable` in the :ref:`product form <products/product-form>`.
 
 Inventory
 ---------
 
-In the :guilabel:`Inventory Defaults` sub-section, you can select the eCommerce selling strategy of
-products:
-
-- :guilabel:`Warehouse`: if you have multiple warehouses, you can define the warehouse associated to
-  your website. If you have multiple websites, you can select a different one per website;
-- :guilabel:`Out-of-Stock (Continue Selling)`: enabling it allows customers to continue placing
-  orders even when the product is **out-of-stock**.
-  Leave it  unchecked to **prevent orders**;
-- :guilabel:`Show Available Qty`: enabling it displays the available quantity left under a specified
-  threshold on the product page. The available quantity is calculated based on the 'On hand'
-  quantity minus the quantity already reserved for outgoing transfers.
-
-.. seealso::
-   :ref:`Allow only selected customers to buy <cart/prevent-sale>`
+In the :guilabel:`Inventory Defaults` sub-section, select your eCommerce selling strategy:
 
-Selling as kit
---------------
-
-If you are selling non-prepackaged kits (i.e., the kits are made of individual products), we
-recommend you read the related documentation to keep track of your stock.
+- :guilabel:`Out-of-Stock`: enable :guilabel:`Continue Selling` to allow customers to continue
+  placing orders even when the product is **out-of-stock**. Leave the box unchecked to
+  **prevent orders**;
+- :guilabel:`Show Available Qty`: displays the available quantity left under a specified threshold
+  on the product page. The available quantity is calculated based on the 'On hand' quantity minus
+  the quantity already reserved for outgoing transfers.
 
-.. seealso::
-   :doc:`../../inventory_and_mrp/manufacturing/advanced_configuration/kit_shipping`
+.. _products/product-comparison:
 
 Product comparison
 ==================
 
-You can enable a **product comparison tool** for your eCommerce by going to
-:menuselection:`Website --> Configuration --> Settings --> Shop - Products`, and ticking
-:guilabel:`Product Comparison Tool`. This tool allows to save products' **specifications** and
-compare them against each other on a single page.
-
-On the product page, scroll down to the :guilabel:`Specifications` section and click
-:guilabel:`Compare`. Repeat the same process for all products you wish to compare. Then, click the
-:guilabel:`Compare` button of the pop-up window at the bottom of the page to reach the comparison
-summary.
+Your visitors can compare products based on their attributes when the :guilabel:`Product Comparison
+Tool` is enabled. You can activate it by going to :menuselection:`Website --> Configuration -->
+Settings --> Shop - Products` section.
 
 .. note::
    The :guilabel:`Product Comparison Tool` can only be used if :doc:`attributes <products/variants>`
-   are set on the **product's template**.
+   are set on the :ref:`product form <products/product-form>`.
```

Comment:
- I thought this doc page was going to be removed? which would make more sense since you talked about product variants on this doc page here? 
- I would rephrase the second part of the sentence a little to make it more clear: The :guilabel:`Product Comparison Tool` is only available for products with attributes (with a link to the relevant section on this doc).
- I know the option is activated by default when you enable the Product comparison tool, but still think it would be helpful to add a link to the Customozation options somehow in this note (using a short sentence) just in case.
-